src/helpers/RefreshToken.ts

```diff
@@ -3,11 +3,21 @@ import { AuthProvider, StaticAuthProvider } from 'twitch/lib'
 import { error } from 'console'
 require('dotenv').config()
 
-export default async (): Promise<AuthProvider> => {
-  const clientId = process.env.USER_ID || ''
-  const clientSecret = process.env.SECRET || ''
-  const refreshToken = process.env.REFRESH_TOKEN || ''
+const client_id = process.env.USER_ID || ''
+const client_secret = process.env.SECRET || ''
+const refresh_token = process.env.REFRESH_TOKEN || ''
 
+interface IAuthRequest {
+  clientId: string
+  clientSecret: string
+  refreshToken: string
+}
+
+export default async ({
+  clientId = client_id,
+  clientSecret = client_secret,
+  refreshToken = refresh_token,
+}: IAuthRequest): Promise<AuthProvider> => {
```

Comment:
I think you need to have this be ?: IAuthRequest

Reply:
NVM, tried it on local, didn't seem to work since you're assigning default values. I think making this not be an interface and instead 3 separate arguments might be the best solution. 
```ts
export default async (
  clientId?: string = client_id,
  clientSecret?:string = client_secret,
  refreshToken?:string = refresh_token,
): Promise<AuthProvider> => 
```

Reply:
so that fixed it, then 20 tests broke, f